src/towncrier/newsfragments/394.feature.rst

```diff
@@ -0,0 +1 @@
+Implement Mercurial support
```

Comment:
Just a small suggestion to have this read as a release note text and not as a PR description.

```suggestion
Support was added for Mercurial SCM.
```